markhor_flippers/src/markhor_flippers.cpp

```diff
@@ -26,21 +23,21 @@ static int multiplicator = 0;
 void joyCallback(const sensor_msgs::Joy::ConstPtr& joy)
 {
   std_msgs::Float64 msg;
-  if (flipper_mode_front == true)
-  {
-    msg.data = joy->axes[4] * multiplicator;
-    flipper_fl_pub.publish(msg);
-    flipper_fr_pub.publish(msg);
-  }
-  if (flipper_mode_back == true)
+  if (individual_mode == false)
   {
-    msg.data = joy->axes[4] * multiplicator;
-    flipper_rl_pub.publish(msg);
-    flipper_rr_pub.publish(msg);
-  }
-  if (individual_mode == true)
-  {
-
+    if (flipper_mode_fl == true && flipper_mode_fl ==true)
+    {
+      msg.data = joy->axes[4] * multiplicator;
+      flipper_fl_pub.publish(msg);
+      flipper_fr_pub.publish(msg);
+    }
+    if (flipper_mode_rl == true && flipper_mode_rr == true)
+    {
+      msg.data = joy->axes[4] * multiplicator;
+      flipper_rl_pub.publish(msg);
+      flipper_rr_pub.publish(msg);
+    }
+  } else {
```

Comment:
Personellement j'enleverais toute reference au individual mode, la nouvelle implementation fait qu'on a pas vraiment besoin de spécifier le individual mode, on fait juste enable/disable les flippers individuellement

Brief : remove the if, keep the else

Reply:
Humm but what happens if there's individual mode enabled (assuming it's like the back flippers) and then we enable the front flippers. Wouldn't that cause trouble ? Could we also add something like disable all the individual flippers that are not in the more that we want when enable and disable the front or back flippers ?